site/docs/global-api/global-layer.md

```diff
@@ -0,0 +1,81 @@
+---
+title: globalLayer
+parent: global-api
+---
+
+# globalLayer
+
+Creates a globally scoped [layer].
+
+```ts compiled
+// reset.css.ts
+
+import { globalLayer, style } from '@vanilla-extract/css';
```

Comment:
Here, and in a few of the other code blocks, `style` is imported but unused.

Reply:
I have a TODO item to lint JavaScript code blocks in Markdown using https://github.com/eslint/eslint-plugin-markdown, but I didn't want to overload this already heavy PR.